exonum-java-binding-qa-service/src/test/java/com/exonum/binding/qaservice/QaServiceImplIntegrationTest.java

```diff
@@ -272,6 +276,28 @@ void getValueBeforeInit() {
     );
   }
 
+  @Test
+  void afterCommit() throws CloseFailuresException {
+    try (MemoryDb db = MemoryDb.newInstance();
+        Cleaner cleaner = new Cleaner()) {
+      Fork view = db.createFork(cleaner);
+      service.initialize(view);
+
+      BlockCommittedEvent event = mock(BlockCommittedEvent.class);
+      service.afterCommit(event);
+
+      // Check that the initial counter was updated.
+      QaSchema schema = new QaSchema(view);
+      MapIndex<HashCode, Long> counters = schema.counters();
+      MapIndex<HashCode, String> counterNames = schema.counterNames();
```

Comment:
What is the reason behind having counter names?

Reply:
Below we check that our `after_commit_counter` counter has correct name.
Or do you mean why we don't have HashCodes instead of names in QA service as a whole? That I don't know, probably it's just easier to create counters that way